lib/node_modules/@stdlib/array/base/linspace2d/docs/types/index.d.ts

```diff
@@ -0,0 +1,51 @@
+/*
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+// TypeScript Version: 4.1
+
+import { Shape2D } from '@stdlib/types/ndarray';
+
+/**
+* Two-dimensional nested array.
+*/
+type Array2D<T> = Array<Array<T>>;
+
+/**
+* Generates a linearly spaced two-dimensional nested numeric array.
+*
+* @param start - first array value
+* @param stop - last array value
+* @param shape - array shape
+* @param colexicographic - specifies whether the array values should be in colexicographic order
+* @returns linearly spaced two-dimensional nested numeric array
+*
+* @example
+* var linspace2d = require( '@stdlib/array/base/linspace2d' );
+*
```

Comment:
```suggestion
```

I am not sure where this practice of requiring the package in TS declarations started, but this needs to be fixed.